composables/drop/massmint/useDropMassMint.ts

```diff
@@ -18,6 +18,7 @@ export type MassMintNFT = Omit<ToMintNft, 'priceUSD'> & {
   hash: string
   sn?: number
   entropyRange: EntropyRange
+  canRender: boolean
```

Comment:
Is there any simpler way than this?
Is it possible to render a static image if multiple amounts? Else, render canvas

Reply:
hmmm , correct me if I'm wrong , but as far I know we need to render the canvas at least once to be able to upload the metadata. 

we don't have access to any static image of that nft variation, unless we have them pre-generated. (iirc that was mentioned as a possibility to speed up our minting process )

Reply:
> hmmm , correct me if I'm wrong , but as far I know we need to render the canvas at least once to be able to upload the metadata.

aahh, I see 👍🏻
let me re-check

Reply:
We can get base64 static image from `toMintNft.imageDataPayload?.image`. Could you check if it is still laggy if we load multiple base64 images?

Reply:
checking

Reply:
added cb48a04

## the break 

 after 2-3 canvas i does hang up for a couple of  secs 

 i'm running a m1 pro ( better than the average user's computer (I would assume)) and I see my CPU usage going crazy 

I can image some user's browser just crashing/ not responding ,  , lowering the max will make for slower minting process  , not sure how do we improve this

## overall 

I do see a difference , it's hard to see on these videos

30 nfts

canary

even though they rendered faster (no max amount) it's kinda laggy and I see some nfts going blank (there is memory cap or something?)   

https://github.com/kodadot/nft-gallery/assets/44554284/35977900-5007-4b70-8742-a80381051517

this pr 

renders slower ,(max of 3 at once) but after that its pretty smooth. 

https://github.com/kodadot/nft-gallery/assets/44554284/6b6b6ed1-fc25-45e9-9258-277ffc4d82c8

Reply:
> renders slower ,(max of 3 at once) but after that its pretty smooth.

nice, is it still faster if we remove "max of 3" stuff?

Reply:
would be nice if we can get rid of `isRendering` stuff